documentation/configuration/parameters/vaultSecrets.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: documentation
+menu: configuration
+pill: vaultSecrets
+subtitle: flyway.vaultSecrets
+---
+
+# Vault Secrets
+{% include teams.html %}
```

Comment:
We should refer to it as 'HasiCorp Vault' since 'Vault' by itself doesn't mean much. We should also link to the Vault docs to give more context.